src/game/items.c

```diff
@@ -637,6 +645,36 @@ bool Item_GetAnimChange(ITEM_INFO *item, ANIM_STRUCT *anim)
     return false;
 }
 
+void Item_PlayAnimSFX(ITEM_INFO *item, int16_t *command, uint16_t flags)
+{
+    if (item->frame_number != command[0]) {
+        return;
+    }
+
+    SOUND_PLAY_MODE mode;
+    switch (command[1] & 0xC000) {
```

Comment:
Should we use constants for `0xC000`, `0x4000`, and `0x8000`?

Reply:
Thanks 👍 I've tidied it up a bit, and managed to get rid of the switch statement in the process.